docs/regenerating-ssl-certificates.md

```diff
@@ -0,0 +1,103 @@
+---
+id: regenerating-ssl-certificates
+title: Regenerating SSL Certificates
+---
```

Comment:
Did you want to add an description?  I don’t think the default is the comment.

Reply:
Yes.  Good catch.  Thanks.